src/main/java/br/com/deveficiente/calendario/controller/form/EventoForm.java

```diff
@@ -17,6 +21,9 @@
     private LocalDateTime inicio;
     @Future
     private LocalDateTime fim;
+
+    private Long codAgenda;
+    @NotNull @NotEmpty(message = "Adicione pelo menos um convidado para criar o evento")
     private List<String> convidados;
```

Comment:
Duas coisas importantes :). Sempre que puder, já inicie a variável com algum valor. No caso da lista, você já pode deixar um new ArrayList, assim ninguém corre risco de tomar NPE. Claro que tem variáveis que não tem valor para começar :). 

*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

A segunda é mais legal :). A String precisa ser um email, da para você fazer assim: ```List<@Email String> convidados```. Você também já pode colocar um @Size(min=1) para garantir pelo menos um convidado. Aí você só fica com a validação de emails que existem :).